AxonDeepSeg/visualization/get_masks.py

```diff
@@ -17,17 +21,16 @@ def get_masks(path_prediction):
     myelin_prediction = prediction > 100
     myelin_prediction = myelin_prediction ^ axon_prediction
 
-    # get main path
-    path_folder, file_name = os.path.split(path_prediction)
-
-    tmp_path = path_prediction.split('_seg-axonmyelin')
-
-    # save the masks
-    if tmp_path[0].endswith('.png'):
-        tmp_path[0] = os.path.splitext(tmp_path[0])[0]
+    # We want to keep the filename path up to the '_seg-axonmyelin' part
+    path = path_prediction.parent
```

Comment:
We should use a more specific variable name than `path` (used to be `path_folder`?). Because `path` is also the name of a module (i.e. try `import path`), which may have been imported, so this would behave strange, and it reads off later on (I though line 32 below was calling the module, not a method of this object).